### ISSUE TITLE
New release request

### ISSUE THREAD

**Issue by viraptor** [cucumber/cucumber-rails#375]
## Summary

Could you provide a new official release, please?
There hasn't been one in a number of months, but there are some changes in master that fix the deprecation warning. It would be great to have them available in a local CI.

**Comment by xtrasimplicity** [comment]
@mvz, @olle - are you guys happy for me to prepare a patch release?

**Comment by Kosmas** [comment]
@xtrasimplicity can you please try to increase the capybara dependency too in the gemspec (i.e < 4) :-)

**Comment by viraptor** [comment]
@xtrasimplicity Yes, that's what we're going to do most likely, but it comes with small annoyances. (slow bundle install, no gem cache, etc.) Thanks for checking though, I'll be watching out for updates.

**Comment by triskweline** [comment]
Thank you for your work @xtrasimplicity @mvz :hearts:

**Comment by xtrasimplicity** [comment]
Not a problem at all! :)
